.vscode/extensions.json

```diff
@@ -12,10 +12,6 @@
     // https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
     "EditorConfig.EditorConfig",
 
-    // Bracket Pair Colorizer 2
```

Comment:
This is now "built-in" to VS Code and if you try to install it as a separate extension it fusses at you. [This affects all of our labs](https://github.com/UMM-CSci-3601/3601-iteration-template/issues/594) and we'll have to make changes in those as well.